exec-unittests

```diff
@@ -167,41 +176,55 @@ _para_jobs_file="${_toxenvdir}/agent-legacy-jobs.lis"
 trap "rm -f ${_para_jobs_file}" EXIT INT TERM
 
 for _major in ${major_list}; do
-    # Verify the Agent or Server Makefile functions correctly.
-    if [[ "${_major}" == "agent" || "${_major}" == "server" ]]; then
-        verify_make_source_tree ${_major} || rc=1
-    fi
-
     if [[ "${_major}" == "agent" ]]; then
-        # The parallel program is really cool.  The usage of `parallel` is
-        # internal and automated; only test code depends on this tool, and we,
-        # as developers, have viewed the citation and are justified in
-        # suppressing future displays of it in our development processes (use of
-        # --will-cite below).
-
-        let count=0
+        if [[ "${subtst:-legacy}" == "python" ]]; then
+            continue
+        fi
```

Comment:
The default value is cool an' all, but it seems unnecessary:  if `subtst` is undefined or blank, it won't be equal to `python` anyway, so replacing it with `legacy` doesn't accomplish much (other than making me ponder why we're doing it 😉).  So, I suggest comparing `"${subtst}"` "as is" to `"python"`.

Ditto line 210.

(And, as long as we're giggling over this stuff, the quotation marks around `"python"` don't accomplish anything either -- we've just been programming in languages like C and Python for so long that they _seem_ like they are required.  😆)

Reply:
I guessed this would come up.  =)

The reason I used the defaulting was to help me understand the flow through this.  We are being a bit tricky by allowing the `subtst` variable to remain empty to indicate a "do it all" behavior, but at times we need to default the value to make it easier to know what to do in a given case.

So I kept the defaulting in place for the expected context for the operation being considered.

Reply:
I get the part about having "empty" mean "all", but I don't see how applying a default of `legacy` here helps provide clarity.  (In fact, I'm inclined to argue that it does the opposite.)

I think what we're trying to express in this code is that, if `${subtst}` is, specifically, `python` and nothing else, then we should skip to the next iteration.  Saying first that if `${subtst}` is empty/all then treat it as though it were `legacy` (so that it won't match `python`) just makes the reader wonder, since empty already isn't `python`.

If you really want to apply a default here, how about using `all`, which would be semantically consistent with the intended use of empty, and that would fail to match `python` just as effectively as `legacy` would....

Reply:
I don't see the value in that, nor do I believe it is confusing.

Since this is causing this PR to be delayed, I'll just remove its use.